test/ArcxAnalyticsProvider.test.tsx

```diff
@@ -152,35 +158,30 @@ describe('(int) ArcxAnalyticxProvider', () => {
       )
       expect(await screen.findByText(`Identity: ${TEST_IDENTITY}`)).to.exist
 
-      expect(postRequestStub.calledTwice).to.be.true
-      expect(
-        postRequestStub
-          .getCall(0)
-          .calledWithExactly(DEFAULT_SDK_CONFIG.url, TEST_API_KEY, '/identify'),
-      ).to.be.true
-      expect(
-        postRequestStub
-          .getCall(1)
-          .calledWithExactly(DEFAULT_SDK_CONFIG.url, TEST_API_KEY, '/submit-event', {
-            identityId: TEST_IDENTITY,
-            event: FIRST_PAGE_VISIT,
-            attributes: {
-              utm: {
-                source: TEST_UTM_SOURCE,
-                medium: TEST_UTM_MEDIUM,
-                campaign: TEST_UTM_CAMPAIGN,
-              },
-            },
-          }),
-      ).to.be.true
+      expect(postRequestStub).calledOnceWithExactly(
+        DEFAULT_SDK_CONFIG.url,
+        TEST_API_KEY,
+        '/identify',
+      )
+
+      fail('how can we test the FIRST_PAGE_VISIT if it is in the callback of connect?')
```

Comment:
@xitronix this is the only spot that is left. Do you have any idea on how we can test it given that now the FIRST_PAGE_VISIT is in the callback?